lib/src/main/java/com/thenewboston/api/common/GetDataSource.kt

```diff
@@ -160,6 +159,19 @@ class GetDataSource @Inject constructor(private val networkClient: NetworkClient
         }
     }
 
+    suspend fun bankConfirmationServices(pagination: PaginationOptions): Outcome<ConfirmationServicesList> {
```

Comment:
Let's rename this function `confirmationServices` Since this is used by both the bank and validator API.